blivet/formats/__init__.py

```diff
@@ -46,13 +46,13 @@ def register_device_format(fmt_class):
     log.debug("registered device format class %s as %s", fmt_class.__name__,
                                                          fmt_class._type)
 
-default_fstypes = ("ext4", "ext3", "ext2")
+default_fstypes = ("ext4", "xfs", "ext3", "ext2")
```

Comment:
Since *xfs* is the default fstype in RHEL7 now, I think it should go to the first place here.